crates/astria-bridge-withdrawer/src/bridge_withdrawer/submitter/signer/frost_signer.rs

```
@@ -0,0 +1,312 @@
+                     commitment,
+                     ..
+                 }| CommitmentWithIdentifier {
+                    commitment: commitment.clone(),
+                    participant_identifier: id.serialize().into(),
+                },
+            )
+            .collect();
+        for Round1Response {
+            id,
+            request_identifier,
+            ..
+        } in responses
+        {
+            let mut client = self
+                .participant_clients
+                .get(&id)
+                .expect(
+                    "participant client must exist in mapping, as we received a commitment from \
+                     them in part 1, meaning we already have their client",
+                )
+                .clone();
+            let request_commitments = request_commitments.clone();
+            let tx_bytes = tx_bytes.clone();
+            stream.push(async move {
+                let resp = client
+                    .execute_round_two(RoundTwoRequest {
+                        request_identifier,
+                        message: tx_bytes.into(),
+                        commitments: request_commitments,
+                    })
+                    .await
+                    .wrap_err(format!(
+                        "failed to get part 2 response for participant with id {id:?}"
+                    ))?;
+                Ok((id, resp.into_inner()))
+            });
+        }
+        let results: Vec<eyre::Result<_>> = stream.collect::<Vec<_>>().await;
+        let sig_shares: BTreeMap<Identifier, frost_ed25519::round2::SignatureShare> = results
+            .into_iter()
+            .filter_map(|res| match res {
+                Ok((id, part2)) => {
+                    let sig_share =
+                        frost_ed25519::round2::SignatureShare::deserialize(&part2.signature_share)
+                            .ok()?;
+                    Some((id, sig_share))
+                }
+                Err(_) => None,
```

Silently dropping the error.